src/gsumm.c

```diff
@@ -726,22 +726,26 @@ SEXP gmin(SEXP x, SEXP narm)
     break;
   case STRSXP:
     ans = PROTECT(allocVector(STRSXP, ngrp)); protecti++;
+    for (i=0; i<ngrp; i++) SET_STRING_ELT(ans, i, NA_STRING);
     if (!LOGICAL(narm)[0]) {
-      for (i=0; i<ngrp; i++) SET_STRING_ELT(ans, i, R_BlankString);
+      Rboolean *upd = calloc(ngrp, sizeof(Rboolean));
```

Comment:
this should be PROTECT(allocateVector(LGLSXP, ngrp)) IINM. then UNPROTECT(1) instead of free below

Reply:
Rboolean is not LGLSXP so not need protect, it is just int of 0,1,INT_MIN. I prefer to use bool instead.

Reply:
@jangorecki  I've just changed it to bool. The INT_MIN is not happening here.